### ISSUE TITLE
[php-7.0-rc8] php-config shows binary path as NONE

### ISSUE THREAD

**Issue by pinepain** [oerdnj/deb.sury.org#142]
Typical symptom - can't run make test on php extensions:

```
$ make test

Build complete.
Don't forget to run 'make test'.

ERROR: Cannot run tests without CLI sapi.
```

More details:

```
$ which php
/usr/bin/php
```
```
$ php -v
PHP 7.0.0RC8 (cli) ( NTS )
Copyright (c) 1997-2015 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2015 Zend Technologies
```

```
$ php-config --php-binary
NONE
```

```
$ php-config
Usage: /usr/bin/php-config [OPTION]
Options:
  --prefix            [/usr]
  --includes          [-I/usr/include/php/20151012 -I/usr/include/php/20151012/main -I/usr/include/php/20151012/TSRM -I/usr/include/php/20151012/Zend -I/usr/include/php/20151012/ext -I/usr/include/php/20151012/ext/date/lib ]
  --ldflags           [-L/usr/lib/php/20151012  -L/usr/lib/x86_64-linux-gnu/mit-krb5]
  --libs              [-lcrypt   -lz -lexslt -lresolv -lcrypt -ledit -ltermcap -lmcrypt -lltdl -lstdc++ -lgmp -lenchant -ldb -lqdbm -lz -lrt -lm -ldl -lnsl  -lxml2 -lgssapi_krb5 -lkrb5 -lk5crypto -lcom_err -lssl -lcrypto -lxml2 -lssl -lcrypto -lxml2 -lxml2 -lcrypt -lxml2 -lxml2 -lxml2 -lxslt -lxml2 -lssl -lcrypto -lcrypt ]
  --extension-dir     [/usr/lib/php/20151012]
  --include-dir       [/usr/include/php/20151012]
  --man-dir           [/usr/share/man]
  --php-binary        [NONE]
  --php-sapis         [libapache2-mod-php7.0 libphp7.0-embed php7.0-cgi php7.0-cli php7.0-fpm php7.0-phpdbg ]
  --phpapi            [20151012]
  --configure-options [--includedir=/usr/include --mandir=/usr/share/man --infodir=/usr/share/info --disable-silent-rules --libdir=/usr/lib/x86_64-linux-gnu --libexecdir=/usr/lib/x86_64-linux-gnu --disable-maintainer-mode --disable-dependency-tracking --prefix=/usr --disable-cli --disable-cgi --with-config-file-path=/etc/php/7.0/apache2 --with-config-file-scan-dir=/etc/php/7.0/apache2/conf.d --build=x86_64-linux-gnu --host=x86_64-linux-gnu --libdir=/usr/lib/php --libexecdir=/usr/lib/php --datadir=/usr/share/php/7.0 --program-suffix=7.0 --sysconfdir=/etc --localstatedir=/var --mandir=/usr/share/man --disable-all --disable-debug --disable-rpath --disable-static --with-pic --with-layout=GNU --without-pear --enable-bcmath --enable-calendar --enable-ctype --enable-dba --with-db4=/usr --without-gdbm --with-qdbm=/usr --enable-inifile --enable-flatfile --enable-dom --with-enchant=/usr --enable-exif --with-gettext=/usr --with-gmp=/usr --enable-fileinfo --enable-filter --enable-ftp --enable-hash --with-iconv --enable-json --enable-mbregex --enable-mbregex-backtrack --enable-mbstring --disable-opcache --enable-phar --enable-posix --with-mcrypt=/usr --enable-mysqlnd-compression-support --with-zlib-dir=/usr --with-openssl=yes --with-libedit=/usr --enable-libxml --enable-session --enable-simplexml --enable-soap --enable-sockets --enable-tokenizer --enable-xml --enable-xmlreader --enable-xmlwriter --with-xsl=/usr --with-mhash=yes --enable-sysvmsg --enable-sysvsem --enable-sysvshm --enable-zip --with-system-tzdata --enable-dtrace --enable-pdo --enable-mysqlnd --with-recode=shared,/usr --with-sqlite3=shared,/usr --with-pdo-sqlite=shared,/usr --with-pdo-dblib=shared,/usr --with-gd=shared,/usr --enable-gd-native-ttf --with-jpeg-dir=shared,/usr --with-xpm-dir=shared,/usr/X11R6 --with-png-dir=shared,/usr --with-freetype-dir=shared,/usr --with-vpx-dir=shared,/usr --with-ldap=shared,/usr --with-ldap-sasl=/usr --enable-intl=shared --with-snmp=shared,/usr --with-pgsql=shared,/usr --with-pdo-pgsql=shared,/usr --with-imap=shared,/usr --with-kerberos --with-imap-ssl=yes --with-mysqli=shared,mysqlnd --with-pdo-mysql=shared,mysqlnd --with-tidy=shared,/usr --with-pspell=shared,/usr --with-curl=shared,/usr build_alias=x86_64-linux-gnu host_alias=x86_64-linux-gnu CFLAGS=-g -O2 -fstack-protector-strong -Wformat -Werror=format-security -O2 -Wall -fsigned-char -fno-strict-aliasing -g LDFLAGS=-Wl,-Bsymbolic-functions -Wl,-z,relro CPPFLAGS=-D_FORTIFY_SOURCE=2 CXXFLAGS=-g -O2 -fstack-protector-strong -Wformat -Werror=format-security]
  --version           [7.0.0RC8]
  --vernum            [70000]
```

**Comment by oerdnj** [comment]
JFTR I have briefly look why this is happening and don't see a reason for this behaviour. So I'll have to have a closer look into build logs.

**Comment by oerdnj** [comment]
Fixed as #224

**Comment by pinepain** [comment]
Thanks!
